service/notesservice.php

```diff
@@ -112,6 +112,9 @@ public function update ($id, $content, $userId){
 
         // prevent directory traversal
         $title = str_replace(array('/', '\\'), '',  $title);
+        // remove hash and space characters from the beginning of the filename
+        // in case of markdown
+        $title = ltrim($title, ' #');
```

Comment:
Shouldn’t it be `# ` instead of ` #`? And also ideally trim `#` as well in case someone doesn’t put a space behind the hash.